islandora_videojs.module

```diff
@@ -75,6 +75,15 @@ function template_preprocess_islandora_videojs(array &$variables) {
   if (variable_get('islandora_videojs_responsive', TRUE)) {
     drupal_add_css(drupal_get_path('module', 'islandora_videojs') . '/css/videojs_responsive.css', array('group' => CSS_DEFAULT, 'every_page' => FALSE));
   }
+  $object = islandora_object_load($variables['params']['pid']);
```

Comment:
Something sits odd with me, about this... the [theme definition](https://github.com/Islandora/islandora_videojs/pull/53/files#diff-1c1ee2d3bd6211e965152b6e81e3f8df6eafbd96332ebe492907732c056d5651R47) above indicates that we allow/expect the object to be passed... but looking into things, it looks like it wouldn't be... though looks like it could easily be made to be passed in the [callback](https://github.com/Islandora/islandora_videojs/pull/53/files#diff-1c1ee2d3bd6211e965152b6e81e3f8df6eafbd96332ebe492907732c056d5651R95-R97) below... to make it something like:

```
function islandora_videojs_callback($params = array(), $object = NULL) {
  return theme('islandora_videojs', array('params' => $params, 'object' => $object));
}
```

As in, the [parameters for the callback](https://github.com/Islandora/islandora/blob/c34c7ac071ff0ac6802ea8701cb38a9f8ed49f76/includes/solution_packs.inc#L797) passes the object, our implementation here just doesn't touch it.

... if this change was made, we would then be able to access the object as `$variables['object']` here, instead of trying to do the load.

... that said, could live with this, as-is.

Reply:
I think I'd prefer the "live with this" approach if it's OK with you.